framework/db/ColumnSchemaBuilder.php

```diff
@@ -270,6 +290,25 @@ public function append($sql)
     }
 
     /**
+     * Sets a foreign key for the column
+     * @param string $foreignKeyName the name of the foreign key constraint.
+     * @param string $refTable the table that the foreign key references to.
+     * @param string $refColumn the name of the column that the foreign key references to.
+     * @param string $delete the ON DELETE option. Most DBMS support these options: RESTRICT, CASCADE, NO ACTION, SET DEFAULT, SET NULL
+     * @param string $update the ON UPDATE option. Most DBMS support these options: RESTRICT, CASCADE, NO ACTION, SET DEFAULT, SET NULL
+     * @return $this
+     */
+    public function foreignKey($foreignKeyName, $refTable, $refColumn, $delete=null, $update=null)
```

Comment:
this name clashes with an existing functionality on the generator http://www.yiiframework.com/doc-2.0/guide-db-migrations.html#foreign-keys